src/libostree/ostree-repo-pull.c

```diff
@@ -4970,6 +4970,8 @@ ostree_repo_find_remotes_async (OstreeRepo                     *self,
   g_autoptr(GTask) task = NULL;
   g_autoptr(FindRemotesData) data = NULL;
   OstreeRepoFinder *default_finders[4] = { NULL, };
+  OstreeRepoFinder *sanitized_finders[4] = { NULL, };
+  guint finder_index = 0;
```

Comment:
Are you sure it would be correct to declare `default_finders`/`sanitized_finders` in the respective blocks? Once they go out of scope wouldn't `finders` point to memory that has been freed?

Reply:
Good point, I didn't think of that.